fields/Cargo.toml

```
@@ -61,6 +59,5 @@ features = [ "derive" ]
 version = "1.0"
 
 [features]
-default = [ "parallel", "snarkvm-utilities/default" ]
-parallel = [ "rayon", "snarkvm-utilities/parallel" ]
+default = [ ]
```

While the parallel flag here should can be removed `snarkvm-utilities/default` has default features unrelated to parallelization, so it should stay.

```suggestion
default = [ "snarkvm-utilities/default" ]
```